packages/p2p/src/components/buy-sell/sort-dropdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Dropdown } from '@deriv/components';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import 'Components/buy-sell/sort-dropdown.scss';
+
+const SortDropdown = () => {
+    const { buy_sell_store } = useStores();
+
+    const [selected_value, setSelectedValue] = React.useState('rate');
+
+    const handleChange = e => {
+        setSelectedValue(e.target.value);
+        buy_sell_store.setItems([]);
+        buy_sell_store.setSortBy(e.target.value);
+        buy_sell_store.loadMoreItems({ startIndex: 0 });
+    };
+
+    return (
+        <Dropdown
+            className='sort-dropdown'
+            classNameLabel='sort-dropdown--label'
+            is_align_text_left
+            list={[
+                { text: localize('Exchange rate (Default)'), value: 'rate' },
+                { text: localize('Completion rate'), value: 'completion' },
+            ]}
+            onChange={handleChange}
+            value={selected_value}
+        />
+    );
+};
+
+export default SortDropdown;
```

Comment:
You can get away with not using `observer` here as we're not observing any observables (we're only calling actions from the store), however, it's perfectly okay to stick to wrapping components in `observer` when using the `useStores` hook.

Reply:
Okay, thanks Aaron! :)